server/data/__testutils/testObjects.ts

```diff
@@ -21,6 +22,83 @@ export const createCaseLoads = ({
   ] as CaseLoad[],
 } = {}): CaseLoad[] => caseLoads
 
+export const createCurrentIncentive = ({
+  level = {
+    code: 'STD',
+    description: 'Standard',
+  },
+}: Partial<CurrentIncentive> = {}): CurrentIncentive => ({ level } as CurrentIncentive)
+
+export const createInmateDetail = ({
```

Comment:
Essentially, we have here the three different kinds of representations we have for the same prisoner, coming from different API endpoints.

Reply:
@tpmcgowan could use this for the moving off NOMIS team about standardising data 😄

Reply:
yeah, the data we need does unfortunately seem to fall between several endpoints that all partially overlap 😵‍💫! I've suggested to Al/Dhiraj that something the orchestration service (or similar) could do is return us a 'VSiP Prisoner' - i.e. cobble together what we need from various sources so we just have one representation of that data in the front-end